src/Cli/dotnet/commands/dotnet-msbuild/MSBuildLogger.cs

```diff
@@ -107,15 +111,47 @@ internal static void FormatAndSend(ITelemetry telemetry, TelemetryEventArgs args
 
                 telemetry.TrackEvent(newEventName, maskedProperties, measurements: null);
             }
+            else if (args.EventName == BuildTelemetryEventName)
+            {
+                var newEventName = $"msbuild/{BuildTelemetryEventName}";
+                Dictionary<string, string> properties = new Dictionary<string, string>(args.Properties);
+                Dictionary<string, double> measurements = new Dictionary<string, double>();
+
+                string[] toBeHashed = new[] { "ProjectPath" };
```

Comment:
the Target property should be hashed as well - users can and do define custom MSBuild targets that could reveal PII